src/App.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable react/function-component-definition */
+import React from 'react';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
```

Comment:
- [ ] You used the `react-router-dom` package, but you haven't installed it. Please consider doing so! 🤓 🔧